src/main/scala/nl/knaw/dans/easy/ingest/Conf.scala

```diff
@@ -29,28 +32,30 @@ class Conf(args: Seq[String], props: PropertiesConfiguration) extends ScallopCon
                 |
                 |Usage:
                 |
-                | $printedName [-u <user> -p <password>] [-f <fcrepo-server>][-i] \\
+                | $printedName [-u <user> -p <password>] [-f <fcrepo-server>] [-i] \\
                 | $indent_____ [<staged-digital-object>... | <staged-digital-object-set>]
                 |
                 |Options:
                 |""".stripMargin)
+  private val s = " provided, please check {{app.home}}/cfg/application.properties"
```

Comment:
name of this value might be confusing to the reader as you have also used `s` in the pattern match on [line 44](https://github.com/DANS-KNAW/easy-ingest/pull/41/files#diff-79bfee1b9ba3798ce350de0651bf6808R44) and [line 51](https://github.com/DANS-KNAW/easy-ingest/pull/41/files#diff-79bfee1b9ba3798ce350de0651bf6808R51). Maybe rename this value to `tailMessage` or something? (*first name I could think of, but you can probably come up with something better...*)